docs/blog/pydantic-v2-alpha.md

```diff
@@ -0,0 +1,182 @@
+we hope you'll find the improvements and additions in Pydantic V2 useful.
+
+If you're new to Pydantic: Pydantic is an open-source Python library that provides powerful data parsing and validation &mdash; 
+including type coercion and useful error messages when typing issues arise &mdash; and settings management capabilities.
+See [the docs](/) for examples of Pydantic at work.
+
+## Getting started with the Pydantic V2 alpha
+
+Your feedback will be a critical part of ensuring that we have made the right tradeoffs with the API changes in V2.
+
+To get started with the Pydantic V2 alpha, install it from PyPI. 
+We recommend using a virtual environment to isolate your testing environment:
+
+```bash
+pip install --pre -U "pydantic>=2.0"
+```
+
+Note that there are still some rough edges and incomplete features, and while trying out the Pydantic V2 alpha releases you may experience errors. 
+We encourage you to try out the alpha releases in a test environment and not in production. 
+Some features are still in development, and we will continue to make changes to the API.
+
+If you do encounter any issues, please [create an issue in GitHub](https://github.com/pydantic/pydantic/issues) using the `bug V2` label. 
+This will help us to actively monitor and track errors, and to continue to improve the library’s performance.
+
+This will be the first of several upcoming alpha releases. As you evaluate our changes and enhancements, 
+we encourage you to share your feedback with us.
+
+Please let us know:
+
+* If you don't like the changes, so we can make sure Pydantic remains a library you enjoy using.
+* If this breaks your usage of Pydantic so we can fix it, or at least describe a migration path.
+
+Thank you for your support, and we look forward to your feedback.
+
+---
+
+## Headlines
+
+Here are some of the most interesting new features in the current Pydantic V2 alpha release. 
+For background on plans behind these features, see the earlier [Pydantic V2 Plan](/blog/pydantic-V2/) blog post.
+
+The biggest change to Pydantic V2 is [`pydantic-core`](https://github.com/pydantic/pydantic-core) &mdash;
+all validation logic has been rewritten in Rust and moved to a separate package, `pydantic-core`.
+This has a number of big advantages:
+
+* **Performance** - Pydantic V2 is 5-50x faster than Pydantic V1.
+* **Safety** - Pydantic V2 should have far fewer bugs long term.
+* **Maintainability** - we're hoping that, long term, V2 is easier to extend without the risk of introducing bugs or breaking changes.
+
+With the use of `pydantic-core`, the majority of the logic in the Pydantic library is dedicated to generating
+"pydantic core schema" &mdash; the schema used define the behaviour of `pydantic-core` validators and serializers.  
+
+### Ready for experimentation
+
+* **BaseModel** - the core of validation in Pydantic V1 remains, albeit with new method names.
+* **dataclasses** - Pydantic dataclasses are improved and ready to test.
```

Comment:
```suggestion
* **Dataclasses** - Pydantic dataclasses are improved and ready to test.
```

The capitalization feels a bit inconsistent here. I recognize the module is called `dataclasses` but I think capitalizing is better here.

Same for "**strict mode**" below — I'd personally use "**Strict mode**"